sdk/test-utils/test-utils/src/chaiAzure.ts

```diff
@@ -1,17 +1,17 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { assert } from "chai";
+import { assert } from "vitest";
 import { OperationTracingOptions } from "@azure/core-tracing";
-import { supportsTracing } from "./tracing/chaiAzureTrace";
+import { supportsTracing } from "./tracing/chaiAzureTrace.js";
 
 /**
  * Augments Chai with support for Azure specific assertions.
  *
  * Sample usage:
  *
  * ```ts
- * import chai from "chai";
+ * import { chai, describe, it, assert } from "vitest";
```

Comment:
is this still the right way to extend vitest's version of chai? Can we switch to augmenting the types a different way by exporting assert from test-utils instead of modifying the chai global definition of assert?

Reply:
@xirzec 

Yes, if we're using chai, that's still the way to do it as far as I can tell: https://vitest.dev/guide/extending-matchers.html

If we were using `vitest` matchers, that would be a different story, but not sure how we can easily do that given the number of assertions required.

Reply:
Removed the chai assertion in favor of the expect.  Maybe we should have both?